core/client/fs/src/main/java/alluxio/client/meta/MetaMasterConfigClient.java

```diff
@@ -75,4 +75,13 @@ void setPathConfiguration(AlluxioURI path, Map<PropertyKey, String> properties)
    * @param path the path
    */
   void removePathConfiguration(AlluxioURI path) throws IOException;
+
+  /**
+   * Updates properties.
+   *
+   * @param propertiesMap the properties map to be updated
+   * @return the update properties status map
+   */
+  Map<PropertyKey, Boolean> updateConfiguration(
+      Map<String, String> propertiesMap) throws IOException;
```

Comment:
Can we change input argument to String to `PropertyKey` too?
```suggestion
      Map< PropertyKey, String> propertiesMap) throws IOException;
```